src/contracts/interfaces/IRWAAToken.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title IRWAAToken
+ * @author Aave
+ * @notice Defines the basic interface for an RWAAToken.
+ */
+interface IRWAAToken {
+  /**
+   * @dev Moves `amount` tokens from `from` to `to`. It does
+   * not use any allowance mechanism, and is only callable by the
+   * transfer role admin.
+   *
+   * Returns a boolean value indicating whether the operation succeeded.
+   *
+   * Emits a {Transfer} event.
+   * @param from The address to transfer from.
+   * @param to The address to transfer to.
+   * @param amount The amount to be transferred.
+   * @return True if the transfer was successful, false otherwise.
+   */
```

Comment:
i d simplify
```suggestion
  /**
   * @notice Transfers an amount of aTokens between two users.
   * @dev It checks for valid HF after the tranfer.
   * @dev Only callable by transfer role admin.
   * @param from The address to transfer from.
   * @param to The address to transfer to.
   * @param amount The amount to be transferred.
   * @return True if the transfer was successful, false otherwise.
   */
```